wcivf/apps/parties/templates/parties/party_list.html

```diff
@@ -1,39 +1,34 @@
 {% extends "base.html" %}
 
-{% load breadcrumb_tags %}
-
 {% block page_title %}UK Political Parties{% endblock page_title %}
 {% block og_title %}UK Political Parties{% endblock og_title %}
 {% block og_description %}List of candidates for UK Political Parties{% endblock og_description %}
 
 
 {% block content %}
-<div class="row">
-<section class="columns large-12">
-  <div class="card">
-    <div class="row">
-      <h2>UK political parties</h2>
-      <p>The following is a list of all political parties represented by candidates in Democracy Club's UK election database (est. 2015). The list includes active, deregistered, and defunct parties.</p>
-        
-       <p>Party names and emblems are taken from the <a href="http://www.electoralcommission.org.uk/">Electoral Commission's register of political parties.</p>
-
-      <ul>
+<nav class="ds-breadcrumbs ds-stack" aria-label="You are here: {{ request.path }}">
```

Comment:
Could be an include

Reply:
> A lot of the comments are showing up as "outdated" but I think that is because they are comments on fields made before your commit to indent everything - so they are still worth checking!
> 
> Also the indentation is still off in places, which makes it hard to review. I've had another look at [djhtml](https://github.com/rtts/djhtml) and had a try out myself locally - I think we can use it to help resolve a lot of the issues. If you have it installed in your env, you can run it on all html files in the repo like this:
> 
> ```
> find ./wcivf/ -name '*.html' | xargs djhtml -i
> ```
> 
> Then I think we should add it as a pre-commit hook - so that then when we make a commit that includes any HTML file changes, it will run on those files before committing. So to get this setup, we need to add `djhtml` to the local.txt requirements, [then follow the steps here](https://github.com/rtts/djhtml#pre-commit-configuration). If you haven't got pre-commit configured already, there are instructions in the `INSTALL.md` file in this repo.
> 
> Also happy to go through this together on a call/live share if you want!

I installed without issue, but the above command to run on all files failed: 

```
find ./wcivf/ -name '*.html' | xargs djhtml -i
xargs: djhtml: No such file or directory
```

However, this worked: 
```
pre-commit run --all-files
black....................................................................Passed
djhtml...................................................................Failed
- hook id: djhtml
- files were modified by this hook

reindented wcivf/apps/parties/templates/parties/party_list.html
1 template has been reindented.
78 templates were already perfect!
```